docs/cheat-sheet.md

```diff
@@ -171,7 +171,7 @@ Supports most options from `detect-secrets scan`
             ]
 ```
 
-#### Fail pre-commit if there are non-audited entries
+#### Fail pre-commit if there are unaudited entries
 
 Fail pre-commit if there are non-auditied entries in baseline file, even if the entries are in files not part of current commit.
```

Comment:
Shouldn't this be **unaudited**  instead of **non-audited** also

Reply:
add below in the testcase asserts....

Reply:
Nice catch! https://github.com/IBM/detect-secrets/pull/91/commits/b3e9574c06bad90c297eee729b110f7597872cfd